src/openstudio_lib/LocationTabView.cpp

```diff
@@ -799,25 +791,25 @@ void LocationView::setDaylightSavingsTime(bool enabled) {
 }
 
 void LocationView::setDstStartDayOfWeekAndMonth(int newWeek, int newDay, int newMonth) {
-  model::RunPeriodControlDaylightSavingTime dst = m_model.getUniqueModelObject<model::RunPeriodControlDaylightSavingTime>();
+  auto dst = m_model.getUniqueModelObject<model::RunPeriodControlDaylightSavingTime>();
 
   dst.setStartDate(NthDayOfWeekInMonth(newWeek), DayOfWeek(newDay), MonthOfYear(newMonth));
 }
 
 void LocationView::setDstStartDate(const QDate& newdate) {
-  model::RunPeriodControlDaylightSavingTime dst = m_model.getUniqueModelObject<model::RunPeriodControlDaylightSavingTime>();
+  auto dst = m_model.getUniqueModelObject<model::RunPeriodControlDaylightSavingTime>();
 
   dst.setStartDate(monthOfYear(newdate.month()), newdate.day());
 }
 
 void LocationView::setDstEndDayOfWeekAndMonth(int newWeek, int newDay, int newMonth) {
-  model::RunPeriodControlDaylightSavingTime dst = m_model.getUniqueModelObject<model::RunPeriodControlDaylightSavingTime>();
+  auto dst = m_model.getUniqueModelObject<model::RunPeriodControlDaylightSavingTime>();
 
   dst.setEndDate(NthDayOfWeekInMonth(newWeek), DayOfWeek(newDay), MonthOfYear(newMonth));
 }
 
 void LocationView::setDstEndDate(const QDate& newdate) {
-  model::RunPeriodControlDaylightSavingTime dst = m_model.getUniqueModelObject<model::RunPeriodControlDaylightSavingTime>();
+  auto dst = m_model.getUniqueModelObject<model::RunPeriodControlDaylightSavingTime>();
```

Comment:
Is this just preferring to use auto?

Reply:
auto if template / cast: https://clang.llvm.org/extra/clang-tidy/checks/modernize-use-auto.html#cast-expressions

I'm not a proponent of (almost) always auto, but `T a = mo.cast<T>()` or `T a = mo.getUniqueModelObject<T>()` is a good candidate.